rest-api-spec/src/main/resources/rest-api-spec/test/get/40_routing.yaml

```diff
@@ -28,7 +28,7 @@
           type:    test
           id:      1
           routing: 5
-          fields:  [_routing]
+          stored_fields:  [_routing]
```

Comment:
I don't recall seeing in the docs that `stored_fields` is how you fetch these metadata fields. I probably just missed it, but can you be sure we mention it and have an example?